pkg/reconciler/cluster/cluster_controller.go

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
would very much prefer to see this not in pkg/reconciler/cluster. This dir is far too crowded already. One dir, one controller, e.g. `pkg/reconciler/workloadclusterheartbeat`.